### PR TITLE
javascript-loops-Kitlong Lai

### PR DESCRIPTION
**Pull request by Kitlonglai**
Screenshot 2023-03-30 at 9 25 04 PM

### CODE REVIEW

File: javascript-loops/main.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+            repeatWord,
+            logEachCharacter,
+            doubleAll,
+            getKeys,
+            getValues
+ */
+function getNumbersToTen() {
+  const numbers = [];
+  let currentNumber = 1;
+  while (currentNumber < 10) {
```
Comment (kurtyazdizadeh) [inline javascript-loops/main.js]: this will be short one iteration

File: javascript-loops/main.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  }
+  return numbers;
+}
+
+console.log('getNumbersToTen: ', getNumbersToTen());
+
+function getEvenNumbersToTwenty() {
+  const evenNumbers = [];
+  let currentNumber = 2;
+  while (currentNumber < 20) {
```
Comment (kurtyazdizadeh) [inline javascript-loops/main.js]: this will also be short by one iteration

**Review by kurtyazdizadeh — CHANGES_REQUESTED** [review]
Couple small changes, almost there though!

**Review by kurtyazdizadeh — APPROVED** [review]
Approved 👍

**Comment by Kitlonglai** [comment]
Screenshot 2023-04-03 at 8 57 45 AM
Screenshot 2023-04-03 at 8 57 51 AM

**Pull request merged**